pandas-stubs/core/frame.pyi

```diff
@@ -558,6 +554,10 @@ class DataFrame(NDFrame, OpsMixin):
         | np_ndarray_bool
         | list[_ScalarOrTupleT],
     ) -> DataFrame: ...
+    @overload
+    def __getitem__(self, idx: Scalar | Hashable) -> Series: ...
+    @overload
+    def __getitem__(self, rows: slice) -> DataFrame: ...
```

Comment:
`slice` is also `Hashable`, so it should probably before the second overload.

If possible it would also be great to rename the argument names - there is no argument named `idx` or `rows`, only `key`

https://github.com/pandas-dev/pandas/blob/d182a3495a95c7067c13690cfe037597c3714d5d/pandas/core/frame.py#L3733

Reply:
Thanks.  I made those 2 changes in https://github.com/pandas-dev/pandas-stubs/pull/661/commits/d300c2d48afa08862901750f8ac13bcfc521b774